src/main.js

```diff
@@ -1024,15 +1023,20 @@ function getValidWindowPosition(state, screen) {
 }
 
 function resizeScreen(screen, browserWindow) {
+  console.log('resize screen called!');
```

Comment:
Do we need to keep all of these `console.log` statements?

Reply:
definitely not, not sure why the linter didn't complain :(